### PR TITLE
Update combobox example with amp-autocomplete usage

### PR DESCRIPTION
**Pull request by caroqliu**
/cc @cvializ

### CODE REVIEW

File: examples/source/interactivity-dynamic-content/ComboBox.html [hunk 1]
```diff
@@ -101,27 +63,23 @@
...
-        First we need to add a data source. Here we're using an inline `amp-state` declaration defining a list of `cities` in the US.
-        For long lists of suggestions, it's possible to fetch `amp-state` from a remote JSON endpoint using the `src` attribute.
+        First we need to add an `amp-autocomplete` element by giving it an input field and a data source. Here we're using an inline `script` declaration defining a list of cities in the US as `items`.
+        For long lists of suggestions, it's possible to fetch a remote JSON endpoint using the `src` attribute. 
+        
+        We will use `filter="token-prefix"` as our way to match on the user input and set `min-characters="0"` so that the combo box will display options even when the input is empty.
       -->
-      <amp-state id="cities">
-        <script type="application/json">[
+      <amp-autocomplete filter="token-prefix" min-characters="0" max-entries="5" on="select:AMP.setState({
```
Comment (cvializ) [inline examples/source/interactivity-dynamic-content/ComboBox.html]: If you use `AMP.setState` you need to keep the amp-bind extension. Is the `city` variable used anywhere? If not you can remove it and keep amp-bind removed.

**Comment by cvializ** [comment]
Wow this is awesome, it really validates the project when you can see how much code can be removed to implement the same functionality!

Screen Shot 2019-04-29 at 14 46 27

**Comment by cvializ** [comment]
/cc @sebastianbenz for review and merge approval.

 Should amphtml contributors be able to add reviewers to PRs on ampproject/docs?

File: examples/source/interactivity-dynamic-content/ComboBox.html (hunk 1 above)
Comment (caroqliu) [reply examples/source/interactivity-dynamic-content/ComboBox.html]: Removed!

**Comment by CrystalOnScript** [comment]
@sebastianbenz friendly ping :)

**Pull request merged**